File: examples/node/ticker.js
var faye = require('../../build');

var endpoint = process.argv[2] || 'http://localhost:8000/bayeux',
    client   = new faye.Client(endpoint),
    n        = 0;

setInterval(function() {
  client.publish('/chat/tick', { n: ++n });
}, 1000);

File: examples/ruby/ticker.rb
require 'rubygems'
require 'bundler/setup'
require 'faye'

EM.run {
  endpoint = ARGV.first || 'http://localhost:9292/bayeux'
  client   = Faye::Client.new(endpoint)
  n        = 0

  EM.add_periodic_timer 1 do
    n += 1
    client.publish('/chat/tick', 'n' => n)
  end
}
